marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -250,14 +326,27 @@ sendWsMessage msg = do
     $ WS.managerWriteOutbound wsManager
     $ WS.SendMessage msg
 
-instance monadMarloweHalogenM :: (ManageMarlowe m) => ManageMarlowe (HalogenM state action slots msg m) where
+instance monadMarloweHalogenM ::
+  ( ManageMarlowe m
+  ) =>
+  ManageMarlowe (HalogenM state action slots msg m) where
   createWallet = lift createWallet
-  followContract walletDetails marloweParams = lift $ followContract walletDetails marloweParams
+  followContract walletDetails marloweParams = lift $ followContract
+    walletDetails
+    marloweParams
   createPendingFollowerApp = lift <<< createPendingFollowerApp
-  followContractWithPendingFollowerApp walletDetails marloweParams followAppId = lift $ followContractWithPendingFollowerApp walletDetails marloweParams followAppId
-  createContract walletDetails roles contract = lift $ createContract walletDetails roles contract
-  applyTransactionInput walletDetails marloweParams transactionInput = lift $ applyTransactionInput walletDetails marloweParams transactionInput
-  redeem walletDetails marloweParams tokenName = lift $ redeem walletDetails marloweParams tokenName
+  followContractWithPendingFollowerApp walletDetails marloweParams followAppId =
```

Comment:
mhmhmh, these line breaks also feel a little bit arbitrary... are these made because of a char limit set somewhere? When possible I prefer the formatter to only do alignment and leave the user the choice on when to break lines. If not possible, not a biggie.

Reply:
It is possible... but I think it's also a double-edge sword.

I like formatters to automatically chop lines that are two long, but this one does not chop _all_ function arguments, so you get kind of wonky stuff like this if it's a big long line.

I think in practice, the best way to deal with it is manually break the first argument it the formatter does this, then it will clean it up:

```
someFunc foo bar baz

// formatted
someFunc foo bar
  baz

// Chop before "foo"
someFunc
  foo bar
  baz

// Format again
someFunc
  foo
  bar
  baz
```

I personally wish it could be configured to just go to the last one directly... this is what more opinionated formatters e.g. `elm-format` and `prettier` do.

Reply:
I am willing to pay the price of some odd line breaks for now though if it means I don't have to deal with any more 300-400 col wide lines in the code! That has always been a major annoyance in the codebase for me.